stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -268,23 +268,14 @@ public void setDefaultPublishableKey(@NonNull @Size(min = 1) String publishableK
         this.defaultPublishableKey = publishableKey;
     }
 
-    private void validateKey(@NonNull @Size(min = 1) String publishableKey)
```

Comment:
Note that these lines are not actually changed. They are moved because I exposed the converter method for testing, and therefore reordered it (so that methods go from public/package/private as we go down the file). I added javadoc as well.